toolz/itertoolz.py

```diff
@@ -214,7 +214,7 @@ def _merge_sorted_binary_key(seqs, key):
         yield val1
 
 
-def interleave(seqs, pass_exceptions=()):
+def interleave(seqs):
```

Comment:
No clue. +1 to dropping it.

On Aug 22, 2016 8:45 PM, "Erik Welch" <notifications@github.com> wrote:

> In toolz/itertoolz.py
> <https://github.com/pytoolz/toolz/pull/325#discussion_r75786209>:
>
> > @@ -214,7 +214,7 @@ def _merge_sorted_binary_key(seqs, key):
> >          yield val1
> >
> >
> > -def interleave(seqs, pass_exceptions=()):
> > +def interleave(seqs):
>
> @mrocklin <https://github.com/mrocklin>, care to chime in? I believe
> pass_exceptions=() was included from the very beginning when you
> consolidated your "utils.py" files from various projects. I know logpy,
> for instance, uses pass_exceptions in the interleave function it defines.
>
> I agree that pass_exceptions seems out of place and not sufficiently
> justified, but @mrocklin <https://github.com/mrocklin> has needed it in
> the past, and I'm generally in favor of maintaining backwards compatibility.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pytoolz/toolz/pull/325/files/1384fb02e32793d2c4ada021e621bac04ef0ed66#r75786209>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AASszD3pq1Zf19jQf6m1sLsS8BH3cWHIks5qikKugaJpZM4JpLjG>
> .
>